Dockerfile

```diff
@@ -25,8 +25,9 @@ RUN apt-get update && \
 ######
 RUN apt-get install -y wget jq && \
     mkdir -p /opt/altv/modules/js-module/ && \
-    wget --no-cache -q -O /opt/altv/modules/js-module/libnode.so.83 https://cdn.altv.mp/js-module/${BRANCH}/x64_linux/modules/js-module/libnode.so.${LIBNODE_VERSION} && \
+    wget --no-cache -q -O /opt/altv/modules/js-module/libnode.so.102 https://cdn.altv.mp/js-module/${BRANCH}/x64_linux/modules/js-module/libnode.so.${LIBNODE_VERSION} && \
```

Comment:
Please replace version with `LIBNODE_VERSION` variable for convenience. I may have forgotten to add this in the past.